### DISCUSSION TITLE
Is there any possibilities for DC to support finder extension in Mac?

### DISCUSSION THREAD

**Discussion by Mickychen00** [doublecmd/doublecmd#1446] [Suggestions]
In MacOS's finder, there are several Finder extensions for OneDrive, GoogleDrive, and Nutstore, which can be utilized for deciding whether to download the folders or files locally when I right-click the folders or files.

<img width="424" alt="image" src="https://github.com/doublecmd/doublecmd/assets/28621570/dfe32e9d-904a-40e4-8af6-5466bd1658e5">

My question is whether DC can support it in Mac. In my Windows's total commander, this function is in good operating.

**Comment by rich2014** [comment]
planned to be implemented in WFX Plugin. 

but it may take a long time.
